homework1/src/ReplaceElements.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Arrays;
+
+public class ReplaceElements {
+    public static void main(String[] args) {
+        Integer[] numbers = {7, 10, 11, 15, 2, 3, 3, 10, 11, 20};
+        int numbers_length = numbers.length / 2;
+
+        for (int position = 0; position < numbers_length; position++) {
+            numbers[position] = numbers[position] + 2;
+            System.out.println(numbers[position]);
+        }
+        System.out.println(Arrays.toString(numbers));
```

Comment:
that's not what you need to do
imagine you have an array [3, 3, 1, 5, 8, 2}
then you want to change 3 or 1 to some other element - 5, 8 or 2
so the result in this case will be something like [8, 8, 1, 5, 8, 2]
if you have some question with this or need an advice - ping me

Reply:
Fixed in commit: f1536a4